src/NzbDrone.Core/Parser/ParsingService.cs

```diff
@@ -50,14 +50,22 @@ public class ParsingService : IParsingService
         public Artist GetArtist(string title)
         {
             var parsedAlbumInfo = Parser.ParseAlbumTitle(title);
+
+            if (parsedAlbumInfo != null && !parsedAlbumInfo.ArtistName.IsNullOrWhiteSpace())
+            {
+                title = parsedAlbumInfo.ArtistName;
+            }
             
-            if (parsedAlbumInfo == null || parsedAlbumInfo.ArtistName.IsNullOrWhiteSpace())
+            Artist artistInfo = null;
+            artistInfo = _artistService.FindByName(title);
```

Comment:
Remove the extra line for the initialization, `Artist artistInfo = _artistService.FindByName(title);`